Sources/Clappr/Classes/Extension/UIImageView+Ext.swift

```diff
@@ -0,0 +1,24 @@
+public extension UIImageView {
+    func download(url: URL) {
+        let request = URLRequest(url: url, cachePolicy: .returnCacheDataElseLoad)
```

Comment:
Can we write a test for this method?

You could easily stub an image URL using `OHHttpStub` and then check if the image was correctly loaded.